components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './LoadingSpinner.module.scss';
+
+const LoadingSpinner = () => {
+  return (
+    <article className={styles.loadingWrapper}>
```

Comment:
I would call this a div or a section, not an article? Not sure the exact usage of an article.